spec/waypoint_spec.js

```diff
@@ -519,6 +519,11 @@ describe('<Waypoint>', function() {
   describe('when the scrollable parent does not have positioning', () => {
```

Comment:
This section could be removed entirely, I think, as we won't rely on being positioned any more.